internal/typeinfo/arginfo_test.go

```diff
@@ -241,5 +245,27 @@ func (s *typeInfoSuite) TestGenerateArgInfoMapError(c *C) {
 
 	_, _, err = argInfo.AllStructOutputs("myMap")
 	c.Assert(err, Not(IsNil))
-	c.Assert(err.Error(), Equals, "columns must be specified for non-struct type")
+	c.Assert(err.Error(), Equals, `cannot use map with asterisk unless columns are specified`)
+
+	_, err = argInfo.InputSlice("myMap")
+	c.Assert(err, Not(IsNil))
```

Comment:
Just saw this, there is a builtin checker called `NotNil`, can we use that instead in all the tests?